package.json

```diff
@@ -7,26 +7,37 @@
     "example": "example"
   },
   "jest": {
-    "testMatch": ["<rootDir>/tests/**/*.js"],
+    "testMatch": [
```

Comment:
this entire file is purely style changes, our prettier setup must be different from yours, we would prefer to keep it as is so could you remove this file from the commit?

Reply:
okay will do